### PR TITLE
Improve documentation for the PathPattern.java

### PR DESCRIPTION
**Pull request by arhont375**
Just noticed issue with documentation. Not sure if it's the bets way to deliver how to write multiple patterns though.
Let me know what would be the better wording

### CODE REVIEW

File: common/src/main/java/com/linecorp/centraldogma/common/PathPattern.java [hunk 1]
```diff
@@ -29,8 +29,8 @@
  *   <li>{@code "*.json"} - all JSON files</li>
  *   <li>{@code "/foo/*.json"} - all JSON files under the directory {@code /foo}</li>
  *   <li><code>"/&#42;/foo.txt"</code> - all files named {@code foo.txt} at the second depth level</li>
- *   <li>{@code "*.json,/bar/*.txt"} - use comma to specify more than one pattern. A file will be matched
- *                                     if <em>any</em> pattern matches.</li>
+ *   <li>{@code "*.json","/bar/*.txt"} - if you have more than 1 pattern you can supply them as {@code varargs} or {@link Iterable}.
```
Comment (jrhee17) [inline common/src/main/java/com/linecorp/centraldogma/common/PathPattern.java:32]: I prefer that we spell out 1

```suggestion
 *   <li>{@code "*.json","/bar/*.txt"} - if you have more than one pattern you can supply them as {@code varargs} or {@link Iterable}. 
```

**Review by jrhee17 — APPROVED** [review]
Looks good to me! Can you sign our CLA?

**Review by ikhoon — APPROVED** [review]
Thanks, @arhont375!

**Review by minwoox — APPROVED** [review]
Thanks, @arhont375! ❤️

**Pull request merged**